shell/browser/api/electron_api_screen.cc

```diff
@@ -68,7 +75,20 @@ Screen::~Screen() {
   screen_->RemoveObserver(this);
 }
 
-gfx::Point Screen::GetCursorScreenPoint() {
+gfx::Point Screen::GetCursorScreenPoint(v8::Isolate* isolate) {
+#if BUILDFLAG(IS_LINUX)
+#if BUILDFLAG(OZONE_PLATFORM_WAYLAND)
```

Comment:
```suggestion
#if defined(USE_OZONE)
```

- `IS_LINUX` check is redundant & can be removed
- An are-we-using-Wayland test shouldn't happen at compile time, since it's a runtime decision whether Ozone will use X11 or Wayland as a backend. But runtime detection requires something like [`HasWaylandDisplay()`](https://github.com/electron/electron/blob/3c63f075bd35333b80f78c462cdba1440b3fec55/shell/browser/electron_browser_main_parts_linux.cc#L22) and IMO its cost (e.g. instantiating a `base::Environment`) would be more expensive than the rest of `GetCursorScreenPoint()` for marginal benefit. So maybe let's keep it at build-time, but check only for Ozone rather than Ozone + Wayland.